src/Utils.js

```diff
@@ -17,10 +17,14 @@ export const getRelativePosition = ({ clientX, clientY }, relativeToElement) =>
     };
 };
 
-export const getPinchMidpoint = ([touch1, touch2]) => ({
-    x: (touch1.clientX + touch2.clientX) / 2,
-    y: (touch1.clientY + touch2.clientY) / 2,
-});
+export const getPinchMidpoint = ([touch1, touch2], image) => {
+    const rect = image.parentNode.getBoundingClientRect();
+
+    return {
+        x: (touch1.clientX + touch2.clientX) / 2 - rect.left,
+        y: (touch1.clientY + touch2.clientY) / 2 - rect.top,
```

Comment:
```suggestion
        x: ((touch1.clientX + touch2.clientX) / 2) - rect.left,
        y: ((touch1.clientY + touch2.clientY) / 2) - rect.top,
```

Use wrapper whenever there can be clash between operations